### PR TITLE
POC - refactor for readability.

### PR DESCRIPTION
**Pull request by Pacane**
@emileber is that better?

### CODE REVIEW

File: lib/pages/artist_page.dart [hunk 1]
```diff
@@ -43,63 +40,65 @@ class ArtistPage extends StatelessWidget {
...
+          : null,
+      artist.youtube != null
+          ? new ListTile(
+              leading: const Icon(FontAwesomeIcons.youtube),
+              title: new Text(artist.youtubeHandle),
+              onTap: () => launch(artist.youtube),
+            )
+          : null,
+      artist.website != null
+          ? new ListTile(
```
Comment (emileber) [inline lib/pages/artist_page.dart]: These `ListTile` look like they could be a reusable component to me. 😛

File: lib/pages/event_page.dart [hunk 2]
```diff
@@ -50,67 +44,71 @@ class EventPage extends StatelessWidget {
...
             ),
           ),
         ],
       ),
     );
   }
+
+  List<Widget> eventDetails(Event event, BuildContext context) {
+    String startTime = dateTimeFormatter.format(event.start);
+    String endTime = event.end == null
```
Comment (emileber) [inline lib/pages/event_page.dart:57]: Nice, I like that this was moved in the relevant scope leaving the build function cleaner.

**Review by emileber — COMMENTED** [review]
I really like it 🎉

File: lib/pages/artist_page.dart (hunk 1 above)
Comment (Pacane) [reply lib/pages/artist_page.dart]: @emileber there you go mah man

**Review by emileber — APPROVED** [review]

**Pull request merged**